src/db/models/packages.ts

```diff
@@ -44,3 +47,29 @@ export const getPackagesByTenantId = async (tenantId: string) => {
         return [];
     }
 };
+
+export const updatePackageStatus = async (packageId: string, status: PackageStatus) => {
+    try {
+        const result = await db
+            .update(packages)
+            .set({ status })
+            .where(eq(packages.id, packageId))
+            .returning();
+
+        return result[0] || null;
+    } catch (error) {
+        console.error("Error updating package status:", error);
+        return null;
+    }
+};
+
+export const removeAllPackages = async () => {
+    try {
+        const result = await db.delete(packages);
+        return result; // Return the number of deleted rows (if applicable)
+    } catch (error) {
+        console.error("Error removing all packages:", error);
+        return null;
+    }
+};
```

Comment:
praise: I like the modular approach you took here with the utility functions, clear naming convention as well